snforge_std/src/byte_array.cairo

```diff
@@ -0,0 +1,7 @@
+fn byte_array_as_felt_array(self: @ByteArray) -> Array<felt252> {
+    let mut serialized = array![core::byte_array::BYTE_ARRAY_MAGIC];
+
+    self.serialize(ref serialized);
+
+    serialized
+}
```

Comment:
Shouldn't it be added to the cairo compiler? 🤔 
cc @piotmag769

Reply:
Jokes on you - it's not xD
<img width="790" alt="image" src="https://github.com/foundry-rs/starknet-foundry/assets/56825108/65dfaebc-611b-4b7c-809f-98bdb7d00087">

Reply:
![Uploading image.png…]()

Reply:
XDDDDDDD